src/containers/tutor-home-page/find-student-block/FindStudentBlock.jsx

```diff
@@ -0,0 +1,62 @@
+import { useCallback, useState } from 'react'
+import { useNavigate } from 'react-router'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+
+import TitleBlock from '~/components/title-block/TitleBlock'
+import SearchIcon from '@mui/icons-material/Search'
+
+import useBreakpoints from '~/hooks/use-breakpoints'
+import { styles } from '~/containers/tutor-home-page/find-student-block/find-student-block.styles'
+import bag from '~/assets/img/tutor-home-page/bag.png'
+import { translationKey } from '~/containers/tutor-home-page/find-student-block/constants'
+import { authRoutes } from '~/router/constants/authRoutes'
+import InputWithIcon from '~/components/input-with-icon/InputWithIcon'
+import AppButton from '~/components/app-button/AppButton'
+
+const FindStudentBlock = () => {
```

Comment:
if this is the similar block to that used on students home page, do it reusable and reuse, no need for duplication